generation/WinSDK/enums.json

```diff
@@ -30632,5 +30669,136 @@
         "field": "dwFlags"
       }
     ]
+  },
+  {
+    "name": "BRUSH_STYLE",
+    "members": [
+      {
+        "name": "BS_SOLID",
+        "value": 0
+      },
+      {
+        "name": "BS_NULL",
+        "value": 1
+      },
+      {
+        "name": "BS_HOLLOW",
+        "value": "BS_NULL"
+      },
+      {
+        "name": "BS_HATCHED",
+        "value": 2
+      },
+      {
+        "name": "BS_PATTERN",
+        "value": 3
+      },
+      {
+        "name": "BS_INDEXED",
+        "value": 4
+      },
+      {
+        "name": "BS_DIBPATTERN",
+        "value": 5
+      },
+      {
+        "name": "BS_DIBPATTERNPT",
+        "value": 6
+      },
+      {
+        "name": "BS_PATTERN8X8",
+        "value": 7
+      },
+      {
+        "name": "BS_DIBPATTERN8X8",
+        "value": 8
+      },
+      {
+        "name": "BS_MONOPATTERN",
+        "value": 9
+      }
+    ],
+    "uses": [
+      {
+        "struct": "tagLOGBRUSH",
```

Comment:
I'm surprised this worked. The struct should be LOGBRUSH and LOGBRUSH2. The "tag" part is what's in the headers, but when we scrape them, we get rid of the tag prefix. (We have tried to get rid of all the "tag" prefixes because it's not what developers use.)